src/components/Settings/PhoneAuthentication.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+	<div class="phone-authentication">
+		<section v-if="step === 0">
+			<h2>Phone number</h2>
+			<form>
+				<legend>
+					Enter a phone number that can be used to verify your identity with a text message or phone call.
+				</legend>
+				<label
+					class="phone-authentication__phone-label"
+					for="phone_input"
+				>
+					Enter your phone number here:
+				</label>
+				<kv-phone-input
+					class="phone-authentication__phone-input"
+					:class="{ 'phone-authentication__phone-input--error' : $v.phoneNumber.$error}"
+					id="phone_input"
+					ref="phoneInput"
+					v-model="phoneNumber"
+					@blur="$v.phoneNumber.$touch"
```

Comment:
@eddieferrer I think I finally figured out how to get Vuelidate errors to only show when you blur the input! So you don't see "this field is required" before you've ever typed anything.